src/main/resources/avro/variantmethods.avdl

```diff
@@ -320,6 +329,75 @@ org.ga4gh.models.Allele getAllele(
   The ID of the `Allele`.
   */
   string id) throws GAException;
+  
+/******************  /alleles/search/path  *********************/
+/** This request maps to the body of `POST /alleles/search/path` as JSON. */
+record SearchAllelesPathRequest {
+  /**
+  If present, look only at `Allele`s in the specified `Dataset`.
+  */
+  union { null, string } datasetId = null; 
+
+  /**
+  If present, look only at `Allele`s used in the specified `VariantSet`.
+  */
+  union { null, string } variantSetId = null; 
+
+  /**
+  Specifies a search algorithm to use, which defines the exact semantics of what
+  it means for an `Allele` to be "relevant" to a `Path`. If the server does not
+  support the sp[ecified algorithm, it will throw an error.
+  */
+  string algorithm;
```

Comment:
Perhaps there should be a method for the server to return a list of supported algorithms, then...